src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalTable.tsx

```diff
@@ -0,0 +1,361 @@
+import React from 'react';
+import { styled } from '@mui/material/styles';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { useTranslation } from 'react-i18next';
+import { useLocale } from 'src/hooks/useLocale';
+import { useDataGridLocaleText } from 'src/hooks/useMuiLocaleText';
+import { percentageFormat } from 'src/lib/intlFormat';
+import { useGoalLineItems } from './useGoalLineItems';
+
+interface MpdGoalRow {
+  line: string;
+  category: string;
+  amount: number;
+  reference: number;
+  percentage?: boolean;
+}
+
+const ministryExpenses = {
+  benefitsCharge: 1910.54,
+  ministryMileage: 85,
+  medicalMileage: 55,
+  medicalExpenses: 210,
+  ministryPartnerDevelopment: 140,
+  communications: 120,
+  entertainment: 110,
+  staffDevelopment: 175,
+  supplies: 45,
+  technology: 90,
+  travel: 200,
+  transfers: 150,
+  other: 75,
+};
+
+const goal = {
+  netMonthlySalary: 8774.25,
+  taxesPercentage: 0.17,
+  rothContributionPercentage: 0.04,
+  traditionalContributionPercentage: 0.06,
+  ministryExpenses,
+};
+
+const StyledDataGrid = styled(DataGrid)(({ theme }) => ({
+  '.MuiDataGrid-columnHeaderTitle': {
+    fontWeight: 'bold',
+  },
+  '.MuiDataGrid-row.bold': {
+    fontWeight: 'bold',
+  },
+  '.MuiDataGrid-row.top-border': {
+    borderTop: '3px solid black',
+  },
+  '.MuiDataGrid-cell.indent': {
+    paddingLeft: theme.spacing(4),
+  },
+  '.MuiDataGrid-columnHeader.reference, .MuiDataGrid-cell.reference': {
+    // Lighten by 90%
+    backgroundColor: `color-mix(in srgb, ${theme.palette.primary.main} 10%, white)`,
+  },
+}));
+
+const formatNumber = (amount: number, locale: string) =>
+  new Intl.NumberFormat(locale, {
+    style: 'currency',
+    currency: 'USD',
+  }).format(amount);
```

Comment:
We don't have this in a lib/shared library?

Reply:
I wanted to show zeros in the numbers, but I could add another option to `currencyFormat`.